<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="20,80">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Color</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderR">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberR"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>G</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderG">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberG"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>B</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderB">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberB"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Box</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>DimX</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderDimX">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberDimX"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>DimY</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderDimY">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberDimY"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>DimZ</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderDimZ">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberDimZ"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Sphere</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="20,60,20">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Radius</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderRaio">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberRaio"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Ellipsoid</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Rx</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderRx">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberRx"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Ry</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderRy">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberRy"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="20,60,20">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Rz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderRz">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberRz"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,90">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Plane</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="10,70,20">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>               Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderZ">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberZ"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Plotter" name="widget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionabreDialogo"/>
    <addaction name="actionsalvar"/>
    <addaction name="actionFechar"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipsoid"/>
   <addaction name="actionCutEllipsoid"/>
  </widget>
  <action name="actionabreDialogo">
   <property name="text">
    <string>Dimensions</string>
   </property>
  </action>
  <action name="actionsalvar">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/voxel.png</normaloff>:/icones/voxel.png</iconset>
   </property>
   <property name="text">
    <string>PutVoxel</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/cutvoxel.png</normaloff>:/icones/cutvoxel.png</iconset>
   </property>
   <property name="text">
    <string>CutVoxel</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/box.png</normaloff>:/icones/box.png</iconset>
   </property>
   <property name="text">
    <string>PutBox</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/cutbox.png</normaloff>:/icones/cutbox.png</iconset>
   </property>
   <property name="text">
    <string>CutBox</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/sphere.png</normaloff>:/icones/sphere.png</iconset>
   </property>
   <property name="text">
    <string>PutSphere</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/cutsphere.png</normaloff>:/icones/cutsphere.png</iconset>
   </property>
   <property name="text">
    <string>CutSphere</string>
   </property>
  </action>
  <action name="actionPutEllipsoid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/ellipsoid.png</normaloff>:/icones/ellipsoid.png</iconset>
   </property>
   <property name="text">
    <string>PutEllipsoid</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/cutellipsoid.png</normaloff>:/icones/cutellipsoid.png</iconset>
   </property>
   <property name="text">
    <string>CutEllipsoid</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>mudaB(int)</slot>
    <slot>mudaG(int)</slot>
    <slot>salvaEscultor()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>783</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberR</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberG</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberB</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberDimX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberDimY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberDimZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRaio</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRx</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRy</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRz</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutVoxel</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutVoxel</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutBox</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutBox</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutSphere</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutSphere</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutEllipsoid</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutEllipsoid</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>capturaAcao(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>mudaB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFechar</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>mudaG(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsalvar</sender>
   <signal>triggered(bool)</signal>
   <receiver>widget</receiver>
   <slot>salvaEscultor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abreDialogEscultor()</slot>
  <slot>capturaAcao(bool)</slot>
 </slots>
</ui>
